### PR TITLE
chore(dev-infra): add Nx 13 migrate script

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

File: libs/sdk/testing/src/lib/test-utility/test-utility.spec.ts [hunk 1]
```diff
@@ -134,9 +134,7 @@ describe('Test utility', () => {
 
     expect(imageUrl).toBe('https://example.com/bg/');
 
-    imageUrl = SkyAppTestUtility.getBackgroundImageUrl(
-      new DebugElement(bgEl, document.body, undefined)
-    );
+    imageUrl = SkyAppTestUtility.getBackgroundImageUrl(new DebugElement(bgEl));
```
Comment (Blackbaud-SteveBrush) [inline libs/sdk/testing/src/lib/test-utility/test-utility.spec.ts:137]: The second and third parameters are optional in Angular 12 (and removed in Angular 13).

File: libs/components/packages/src/schematics/ng-add/ng-add.schematic.spec.ts [hunk 2]
```diff
@@ -58,38 +58,13 @@ describe('ng-add.schematic', () => {
...
-      'karma-jasmine-html-reporter': '~1.7.0',
-      'ng-packagr': '^12.1.1',
-      typescript: '~4.3.5',
-    });
+    expect(packageJson.dependencies).toEqual(
+      expect.objectContaining({
+        '@skyux/layout': '5.0.0',
+        '@skyux/core': 'LATEST',
+        '@skyux/i18n': 'LATEST',
+      })
```
Comment (Blackbaud-SteveBrush) [inline libs/components/packages/src/schematics/ng-add/ng-add.schematic.spec.ts:66]: No need to check all dependencies; only the SKY UX dependencies are affected by the test.

File: libs/components/router/src/lib/modules/link/link-external.directive.spec.ts [hunk 3]
```diff
@@ -115,7 +115,7 @@ describe('SkyAppLinkExternal Directive', () => {
       By.directive(SkyAppLinkExternalDirective)
     );
     expect(directive.attributes['target']).toEqual('_top');
-    expect(directive.properties['href']).toEqual('testUrl/test');
+    expect(directive.properties['href'].endsWith('testUrl/test')).toBeTrue();
```
Comment (Blackbaud-SteveBrush) [inline libs/components/router/src/lib/modules/link/link-external.directive.spec.ts:118]: Angular 13 inserts the localhost URL into the `href` string. Changing the test here to accommodate both Angular 12 and 13.

File: scripts/dev-migrate.ts [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+      await runCommand('npm', ['uninstall', '@nrwl/cli', '@nrwl/tao']);
+
+      await runCommand('npx', ['nx', 'migrate', 'latest']);
+
+      await removeUnwantedMigrations();
+
+      await runCommand('npx', ['nx', 'migrate', '--run-migrations']);
+
+      // Update Nx CLI global install.
+      await runCommand('npm', ['install', '--global', 'nx@latest']);
```
Comment (johnhwhite) [inline scripts/dev-migrate.ts]: This global install failed on my machine when creating the symlink — not sure why, but also wouldn't necessarily expect this step. Can this be removed?

**Review by johnhwhite — COMMENTED** [review]
One thing. Obviously lots to follow up on after this gets run.

File: scripts/dev-migrate.ts (hunk 4 above)
Comment (Blackbaud-SteveBrush) [reply scripts/dev-migrate.ts]: Yeah, that's fine. They recommend `npx` anyway. 👍

**Review by johnhwhite — APPROVED** [review]

**Pull request merged**